src/_components/header/header.module.css

```diff
@@ -0,0 +1,21 @@
+@import url("https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;700;800&display=swap");
```

Comment:
This might need an internet connection no? Once this is merged to main, I'll test the bug again and post an issue if it pops up

Reply:
this goes to the other `@import url(...)` with online links as well